dash/__init__.py

```diff
@@ -9,3 +9,4 @@
 from . import resources  # noqa: F401,E402
 from .version import __version__  # noqa: F401,E402
 from ._callback_context import callback_context  # noqa: F401,E402
+from ._dash_renderer import _js_dist_dependencies, _js_dist  # noqa: F401,E402
```

Comment:
Do we need this import? This file is generally for items we want to expose to end users, these are imported directly by `dash.py` where they're used.

Reply:
Removed in 2aefb7fc.